elastic/kibana
# [RAM] Add new API to allow for bulk untrack alerts using query DSL

## Summary
Resolves: https://github.com/elastic/kibana/issues/174640 

This PR adds a new API: `/internal/alerting/alerts/_bulk_untrack_by_query` that allows active alerts to be untracked using ES query DSL. This PR also allows alerts to be bulk untracked by query using the alerts table. 

![image](https://github.com/elastic/kibana/assets/74562234/242934d1-a034-401f-80b7-2c59473974b8)

### To Test:

1. Create 2 rule that generate alerts
2. Navigate to the alerts table, create a filter using the alerts table search bar to filter 1 of the alerts
3. Select the resulting alert, click "select  all alerts"
4. Select "Selected 1 alert" and click "Mark as Untracked"
5. Assert the alert has been untracked
6. Let the rule run again
7. Assert a new active alert should be created
8. Retry steps 1-7 but with a recovered alert. Assert the recover alert does not get untracked



### Checklist
